build.gradle

```diff
@@ -184,11 +184,10 @@ dependencies {
     generatorsImplementation("com.squareup:javapoet:1.13.0")
 
     // Adventure, for text messages
-    api platform("net.kyori:adventure-bom:4.7.0")
```

Comment:
`api platform("net.kyori:adventure-bom:$adventureVersion")` and then keep the rest the same?

Reply:
Yeah I had this with the BOM originally but it was causing problems developing projects that depend on Minestom. I think there's something else wrong with this build file but I couldn't quite work it out and this seemed to fix it, although it's not the best solution that's for sure.